packages/front-end/components/Experiment/EditDOMMutationsModal.tsx

```diff
@@ -0,0 +1,133 @@
+import { VisualChange } from "back-end/types/visual-changeset";
+import { FC, useCallback, useState } from "react";
+import Code from "@/components/SyntaxHighlighting/Code";
+import Modal from "../Modal";
+
+const EditDOMMutatonsModal: FC<{
+  visualChange: VisualChange;
+  close: () => void;
+  onSave: (newVisualChange: VisualChange) => void;
+}> = ({ close, visualChange, onSave }) => {
+  const [newVisualChange, setNewVisualChange] = useState<VisualChange>(
+    visualChange
+  );
+
+  const deleteCustomJS = useCallback(() => {
+    setNewVisualChange({
+      ...newVisualChange,
+      js: "",
+    });
+  }, [newVisualChange, setNewVisualChange]);
+
+  const deleteGlobalCSS = useCallback(() => {
+    setNewVisualChange({
+      ...newVisualChange,
+      css: "",
+    });
+  }, [newVisualChange, setNewVisualChange]);
+
+  const deleteDOMMutation = useCallback(
+    (index: number) => {
+      setNewVisualChange({
+        ...newVisualChange,
+        domMutations: newVisualChange.domMutations.filter(
+          (_m, i) => i !== index
+        ),
+      });
+    },
+    [newVisualChange, setNewVisualChange]
+  );
+
+  const onSubmit = () => {
+    onSave(newVisualChange);
+  };
+
+  return (
+    <Modal
+      open
+      close={close}
+      size="lg"
+      header="Edit Visual Changes"
+      submit={onSubmit}
+      cta="Save"
+    >
+      <div className="col">
+        <div className="mb-4">
+          <h4>
+            Global CSS
+            {newVisualChange.css ? (
+              <small className="ml-2">
+                <a href="#" className="text-danger" onClick={deleteGlobalCSS}>
+                  delete
```

Comment:
I suppose it's the nature of an MVP escape hatch, but having only "delete" in an edit modal feels a bit odd. I think we could at least change the UI a tiny bit here.

Currently it looks visually like:
`1. delete`
... which visually implies that the title of the change is "delete". Maybe delete should be some sort of "x" or trash icon floating on the right instead?

Reply:
Addressed